public/css/signup.css

```diff
@@ -0,0 +1,41 @@
+.signup{
+    display: flex;
+    /* padding-left: 4rem; */
+   
+    align-content: center;
+}
+
+
+
+form{
+    background-color:#E6E9F8 ;
+    width: 60%;
```

Comment:
You've given the signup form a width of 60% of the containing div, causing layout issues. The input fields have an absolute width of 18rem.